### ISSUE TITLE
Feature Request: Submit solutions in the solver format.

### ISSUE THREAD

**Issue by nlordell** [gnosis/dex-cli#60]
This would help us be able to send hand-crafted solutions in case of emergency!

**Comment by anxolin** [comment]
I agree, also good to have for anyone wanting to submit a solution. For now it's read only, but happy to add a connection, actually I have some code to send a Wallet Connect tx, but I would need to get some time to do it
